index.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * decaffeinate suggestions:
+ * DS101: Remove unnecessary use of Array.from
+ * DS102: Remove unnecessary code created because of implicit returns
+ * DS205: Consider reworking code to avoid use of IIFEs
+ * DS207: Consider shorter variations of null checks
+ * Full docs: https://github.com/decaffeinate/decaffeinate/blob/master/docs/suggestions.md
+ */
+const fs = require('fs');
+const path = require('path');
+
+module.exports = function(robot, scripts) {
+  const scriptsPath = path.resolve(__dirname, 'src');
+  if (fs.existsSync(scriptsPath)) {
+    return (() => {
+      const result = [];
+      for (let script of Array.from(fs.readdirSync(scriptsPath).sort())) {
+        if ((scripts != null) && !Array.from(scripts).includes('*')) {
+          if (Array.from(scripts).includes(script)) { result.push(robot.loadFile(scriptsPath, script)); } else {
+            result.push(undefined);
+          }
+        } else {
+          result.push(robot.loadFile(scriptsPath, script));
+        }
```

Comment:
I think this could be reduced to

``` javascript
return fs.readdirSync(scriptsPath)
    .filter(script => scripts == null || scripts.includes('*') || scripts.includes(script))
    .map(script => robot.loadFile(scriptsPath, script));
```

Don't you think?